.classpath

```diff
@@ -1,28 +1,28 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
or you could leave that for now, just merge to master, I will fix it on Master for the other 3 files: Lexer.java, sym.java as well as parser.java, these three files are guarantee to be generated by jflex and cup and they changes all the time, so no need to have them in repo as well